README.md

```diff
@@ -67,25 +67,72 @@ spec:
     targetSize: 40%
 ```
 
-- **enforcementMode**: specifies how the policy will be enforced during scheduler. Values allowed for this field are: 
+- **enforcementMode**: specifies how the policy will be enforced during scheduler. Values allowed for this field are:
   - **BestEffort** (default): the policy will be enforced as best effort (scorer mode).
   - **Strict**: the policy will be forced during scheduling.
 - **nodeSelector**: selects the nodes where the placement policy will apply on according to action.
 - **podSelector**: identifies which pods this placement policy will apply on
 - **action**: policy placement action that carries the following possible values:
   - **Must**(default): based on the rule below pods must be placed on nodes selected by node selector MustNot: based on the rule pods
   - **MustNot** be placed nodes selected by node selector'
-- **targetSize**: the number or percent of pods that can or cannot be placed on the node. 
+- **targetSize**: the number or percent of pods that can or cannot be placed on the node.
 - **weight**: allows the engine to decide which policy to use when pods match multiple policies.
 
 ### Demo
 
 - Create a [KinD](https://kind.sigs.k8s.io/) cluster with the following config
 
 ```sh
-kind create cluster --config https://raw.githubusercontent.com/Azure/placement-policy-scheduler-plugins/main/test/e2e/kind-config.yaml
+cat <<EOF | kind create cluster --name placement-policy --config=-
```

Comment:
Changing this to line because `kind create cluster` doesn't work with remote urls